bentoml/_internal/frameworks/transformers.py

```diff
@@ -262,12 +262,17 @@ class TransformersRunnable(bentoml.Runnable):
         def __init__(self):
             super().__init__()
 
-            available_gpus = os.getenv("NVIDIA_VISIBLE_DEVICES")
+            available_gpus: str = os.getenv("NVIDIA_VISIBLE_DEVICES")
```

Comment:
Should we support `NVIDIA_VISIBLE_DEVICES=all` or `NVIDIA_VISIBLE_DEVICES=1,2`?

Reply:
My understanding is that breaking down `all` and `1,2` to individual devices is the job of the scheduling strategy.